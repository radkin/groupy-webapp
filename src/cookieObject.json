{
  "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySUQiOiI1ZTYyYWRlMTUwMTRiNTQ0YTUwYWY5NDMiLCJpYXQiOjE1ODg5NzQ1MTF9.3Dzqp7doG8HAQ9Xz2kbO9iW9UMVA03q6cjyxfNsGddw",
  "userID": "5e62ade15014b544a50af943"
}
