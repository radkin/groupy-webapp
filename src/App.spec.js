import Cookies from 'universal-cookie';
import cookieObject from './cookieObject';

const cookies = new Cookies();
const cookieExists = cookies.get('groupy')
const App = require('./App')

test('proper profile data is correct', () => {
  expect(cookieObject).toMatchSnapshot();
  expect(cookieObject.userID).toEqual('5e62ade15014b544a50af943');
});

test('new cookies', () => {
  expect(cookies.HAS_DOCUMENT_COOKIE).toBe(true);
});

test('cookieObject should have properties token and userID', () => {
  expect(cookieObject).toHaveProperty('token');
  expect(cookieObject).toHaveProperty('userID');
});
