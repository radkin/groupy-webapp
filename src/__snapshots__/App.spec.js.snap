// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proper profile data is correct 1`] = `
Object {
  "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySUQiOiI1ZTYyYWRlMTUwMTRiNTQ0YTUwYWY5NDMiLCJpYXQiOjE1ODg5NzQ1MTF9.3Dzqp7doG8HAQ9Xz2kbO9iW9UMVA03q6cjyxfNsGddw",
  "userID": "5e62ade15014b544a50af943",
}
`;
